Allow dropping into interactive pdb shell when hitting exception.  Ad… (#1180)
cocotb/cocotb: documentation/source/building.rst

    If defined, log lines displayed in terminal will be shorter. It will print only
    time, message type (``INFO``, ``WARNING``, ``ERROR``) and log message.

.. envvar:: COCOTB_PDB_ON_EXCEPTION

>    If defined, when a python exception is hit, cocotb will drop into the python debugger.

Reviewer: ```suggestion
   If defined, cocotb will drop into the python debugger if a test fails with an exception.
```
As written it suggest that all exceptions will cause pdb, but its really just the ones that result in test failures

>    If defined, cocotb will drop into the python debugger if a test fails with an exception.

Reviewer 2: nit: python -> Python

Reviewer: ```suggestion
   If defined, cocotb will drop into the Python debugger (:mod:`pdb`) if a test fails with an exception.
```

Reviewer: @imphil, does it let you hit the "commit suggestion" button above? It doesn't let me...

Reviewer 2: no, he most likely disabled "maintainers can update this PR"

Reviewer: Seems odd github would show me this then:

![image](https://user-images.githubusercontent.com/425260/72481191-9aae9480-37f1-11ea-9ae9-be8be201c887.png)

I've opened an issue with github support about this ambiguity

Author: Weird.  I dont't think I disabled that option, and I'm not seeing the box to enable it where it should be.
